webapp-django/crashstats/supersearch/models.py

```diff
@@ -212,11 +233,20 @@ class SuperSearchField(models.SocorroMiddleware):
     def get(self, **kwargs):
         raise NotImplemented()
 
+    def create_field(self, **kwargs):
+        return self.implementation.create_field(**kwargs)
+
+    def update_field(self, **kwargs):
+        return self.implementation.update_field(**kwargs)
+
+    def delete_field(self, **kwargs):
+        return self.implementation.delete_field(**kwargs)
+
```

Comment:
So this means those methods do not use the ``fetch`` method anymore, and thus do not benefit from any of the niceties in our ``models`` base class? Is that OK? 

If it is, then you can simply have this class inherit from the implementation class, and then you don't need to define those bridges.